python/src/gen_gherkin/__init__.py

```diff
@@ -67,6 +72,42 @@ def run_gen_gherkin_behaviors(
     return result
 
 
+def run_gen_dictionary_file(
+    architecture_file: str, output_directory: str
+) -> ExecutionResult:
+    """
+    Generate a Dictionary Steps file from AaC model behavior scenarios.
+
+    Args:
+        architecture_file (str): The YAML file containing the data models from which to generate the dictionary file.
+        output_directory (str): The directory into which the generated dictionary file will be written.
+
+    Returns:
+        The results of the execution of the plugin gen-dictionary-file command.
+    """
+
+    result = ExecutionResult(
+        plugin_name, "gen-dictionary-file", ExecutionStatus.SUCCESS, []
+    )
+
+    dictionary_file_check_result = before_gen_gherkin_behaviors(architecture_file, run_check)
+    if not dictionary_file_check_result.is_success():
+        return dictionary_file_check_result
+    else:
+        result.add_messages(dictionary_file_check_result.messages)
```

Comment:
To maintain name consistency.

```suggestion
    gen_dictionary_file_check_result = before_gen_dictionary_file(architecture_file, run_check)
    if not gen_dictionary_file_check_result.is_success():
        return gen_dictionary_file_check_result
    else:
        result.add_messages(gen_dictionary_file_check_result.messages)
```